packages/bpk-component-select/src/BpkSelect.js

```diff
@@ -15,25 +15,48 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/* @flow */
 
-import PropTypes from 'prop-types';
+import PropTypes, { type Node } from 'prop-types';
 import React from 'react';
 import { cssModules } from 'bpk-react-utils';
 
 import STYLES from './bpk-select.scss';
 
 const getClassName = cssModules(STYLES);
 
-const BpkSelect = props => {
+export type Props = {
+  id: string,
+  name: string,
+  value: string,
+  className: ?string,
+  docked: ?boolean,
+  dockedFirst: ?boolean,
+  dockedLast: ?boolean,
+  dockedMiddle: ?boolean,
+  image: ?() => Node,
+  imageContainerClassName: ?string,
+  large: ?boolean,
+  valid: ?boolean,
+  wrapperClassName: ?string,
+};
+
+const BpkSelect = (props: Props) => {
   const classNames = [getClassName('bpk-select')];
+  const imageContainerClassNames = [getClassName('bpk-select-image-container')];
```

Comment:
Shouldn't this be `bpk-select__image-container`?

Reply:
it's following the same principle as the large config e.g. https://github.com/Skyscanner/backpack/blob/master/packages/bpk-component-select/src/__snapshots__/BpkSelect-test.js.snap#L72

with the `--` rather than `__`

Reply:
Ignore that, I didn't understand your point 😅